internal/pkg/p2p/rumexchange.go

```diff
@@ -62,6 +63,7 @@ func (r *RexService) ConnectRex(ctx context.Context) error {
 			if err != nil {
 				rumexchangelog.Debugf("create network stream err: %s", err)
 				r.peerStatus.Update(p, r.ProtocolId, PROTOCOL_NOT_SUPPORT)
+				return err
```

Comment:
It should not be returned.  We need check all peers in a loop and update their `peerStatus`.

Reply:
I see, so that means `ConnectRex` will always return nil, right?
I will fix this and do a force push after rebasing to main.